sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -158,13 +162,13 @@ def __init__(self, hostname, **kwargs):
         self._connection = None
         self._session = None
         self._link = None
-        self._socket_timeout = False
         self._external_connection = False
         self._cbs_authenticator = None
         self._auth_timeout = kwargs.pop("auth_timeout", DEFAULT_AUTH_TIMEOUT)
         self._mgmt_links = {}
+        self._mgmt_link_lock = threading.Lock()
         self._retry_policy = kwargs.pop("retry_policy", RetryPolicy())
-        self._keep_alive_interval = int(kwargs.get("keep_alive_interval") or 0)
+        self._keep_alive_interval = int(kwargs.get("keep_alive_interval", 0))
```

Comment:
Is the keep-alive interval exposed in EH? I think it used to be?
If so, I think the original syntax might be preferable here.
Otherwise in the case where someone has `keep_alive_interval=None/False`, this would now raise a syntax error.

Reply:
It can be passed in via kwargs though not documented as such. That should also be updated 

Ill change this logic here to handle the None as it was breaking.